protobuf_deps.bzl

```diff
@@ -49,9 +49,9 @@ def protobuf_deps():
         http_archive(
             name = "zlib",
             build_file = "@com_google_protobuf//:third_party/zlib.BUILD",
-            sha256 = "629380c90a77b964d896ed37163f5c3a34f6e6d897311f1df2a7016355c45eff",
-            strip_prefix = "zlib-1.2.11",
-            urls = ["https://github.com/madler/zlib/archive/v1.2.11.tar.gz"],
+            sha256 = "d8688496ea40fb61787500e863cc63c9afcbc524468cedeb478068924eb54932",
```

Comment:
was this change necessary in this PR? this just is a good thing to keep separate typically RE: rollbacks and cherrypicks etc.

Reply:
Yep, this was necessary :). There was a UBSAN bug in zlib that was fixed with 1.2.12